moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -250,6 +250,7 @@ export const ProjectActivityLogTableMaps = {
         label: "partner ID",
         type: "int4",
       },
+      // deprecated column, but keeping because historical activities depend on it
```

Comment:
can we actually go ahead and get rid of this too now? this is the only instance of `partner_name` left in the repo (aside from migrations)

Reply:
We should leave this for the time being, because previous activity log entries may have a reference to this field.